api.yml

```diff
@@ -1,159 +1,62 @@
 openapi: 3.0.0
-# Added by API Auto Mocking Plugin
-servers:
-  - description: SwaggerHub API Auto Mocking
-    url: https://virtserver.swaggerhub.com/danubetech/credential-issuer/0.0.1
 info:
-  description: This is an open API specification for issuing digital credentials based on standards such as the Verifiable Credentials Data Model (https://www.w3.org/TR/vc-data-model/), JWT (https://tools.ietf.org/html/rfc7519), and Open Badges (https://openbadges.org/). Implementations of this API may differ in terms of which data formats or proof formats they do or do not support. Also, implementations may choose to not support certain parts of this API (e.g. optional parts include changing and retrieving the status of a credential, or refreshing a credential).
-  version: "0.0.1"
-  title: Credential Issuer API
+  description: This is an open HTTP API specification for issuing Verifiable Credentials based on the W3C Verifiable Credentials Data Model (https://www.w3.org/TR/vc-data-model/). Implementations of this API may differ in terms of which representation formats or proof formats they do or do not support.
+  version: "0.0.2"
+  title: Verifiable Credential Issuer HTTP API
   contact:
-    email: markus@danubetech.com
+    email: george.aristy@securekey.com
   license:
     name: All documents in this Repository are licensed by contributors under the [W3C Software and Document License](http://www.w3.org/Consortium/Legal/copyright-software).
-tags:
-  - name: internal
-    description: Internal APIs facing the Issuer.
-  - name: public
-    description: Public APIs facing the Holder and Verifier.
 paths:
-  /credential:
+  /credentials/sign:
```

Comment:
```suggestion
  /credentials
```

Reply:
`issue` would be more generic, I'm not sure what the ZKP folks would prefer, but my guess is that they would not really like the term "sign".

Reply:
+1 for keeping these routes under a root path eg. `/credentials`

Reply:
Yes, I don't like "sign" -- the reason being that there is more to issuing a credential than signing it. For example, setting up its revocation management (whether that's done by revocation list or by some fancy ZKP thing).